exts/edgeimpulse.dataingestion/edgeimpulse/dataingestion/extension.py

```diff
@@ -230,15 +230,38 @@ def setup_data_upload_ui(self):
                     self.anomaly_samples_label = ui.Label(
                         f"Anomaly samples: {self.anomaly_samples}"
                     )
+                ui.Spacer(height=10)
+
+                with ui.HStack(height=10):
+                    ui.Spacer(width=3)
+                    ui.Label("Add Bounding Boxes", width=70)
```

Comment:
This makes sense to me as well. However, one issue I have is that I am not sure how to clear the folder path without restarting the whole extension. Because of this, the path for the bounding box data often auto-populates from the last upload and could mess with the logic. 

The checkbox can be a temporary solution to this, instead of checking if there is a path entered. 

"Data path" also seemed too generic to me and could imply many things. I put RGB to align with the output from replicator. But maybe something like "Images Path" could be better?